src/main/java/com/ust/sourcecourse/configuration/exception/CustomException.java

```diff
@@ -0,0 +1,48 @@
+package com.ust.sourcecourse.configuration.exception;
+
+import java.util.List;
+
+public class CustomException extends RuntimeException {
+    /**
+		 * 
+		 */
+		private static final long serialVersionUID = 1L;
+
+	public CustomException(String message) {
+        super(message);
+    }
+
+    public static class ResourceNotFoundException extends CustomException {
```

Comment:
Have each exception classes separately instead of inner classes

Reply:
resolved